### PR TITLE
clean: eslint configs

### PR DESCRIPTION
**Pull request by fontanierh**
- eslint config for `docs/`
- eslint config for `xp1/` (and fix existing violations)

### CODE REVIEW

File: xp1/package.json [hunk 1]
```diff
@@ -1,10 +1,11 @@
 {
-  "name": "Dust XP1",
+  "name": "dust-xp1",
```
Comment (fontanierh) [inline xp1/package.json:2]: Is that OK ?

File: front/.eslintrc.json [hunk 2]
```diff
@@ -5,6 +5,12 @@
     "react-hooks/rules-of-hooks": 0,
     "react-hooks/exhaustive-deps": 0,
     "@next/next/no-img-element": 0,
-    "jsx-a11y/alt-text": 0
+    "jsx-a11y/alt-text": 0,
+    "no-undef": 1
```
Comment (gmpetrov) [inline front/.eslintrc.json:9]: Usually, Typescript projects do not enable this rule as it's already handled by TS

Next is disabling the option too
https://github.com/vercel/next.js/blob/canary/.eslintrc.json#L60

Are you sure you need it?

File: front/tests/lib/datasets.test.ts [hunk 3]
```diff
@@ -1,4 +1,5 @@
 import { checkDatasetData, getDatasetTypes } from "@app/lib/datasets";
+import { describe, test, expect } from "@jest/globals";
```
Comment (gmpetrov) [inline front/tests/lib/datasets.test.ts:2]: Adding the following to .eslintrc.json should be enough when "no-undef" is enabled

"env": {
    "jest": true
}

File: front/.eslintrc.json (hunk 2 above)
Comment (fontanierh) [reply front/.eslintrc.json:9]: If we are committed to write only TS going forward then you are right, we don't need this 👍

File: front/tests/lib/datasets.test.ts (hunk 3 above)
Comment (fontanierh) [reply front/tests/lib/datasets.test.ts:2]: Thank you !

File: xp1/package.json (hunk 1 above)
Comment (spolu) [reply xp1/package.json:2]: It might be picked up by the final extension packaging but we can figure it out later :+1:

**Pull request merged**